### PR TITLE
Homework 19

### PR DESCRIPTION
**Pull request by vladimirchizhenkov**
This code makes my eyes bleed...

### CODE REVIEW

File: homework_16/src/index.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { customChecker } from './custom-checker';
 
-// Positive tests.
+// Positive selenium-tests.
```
Comment (VladSelikh) [inline homework_16/src/index.ts:3]: Why did you change that? Please return back.

File: homework_19/selenium-tests/main.spec.ts [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+        expect(styleDisplay).to.eq('none');
+        expect(cookie.value).to.eq('1');
+    });
+
+    it('Should open and close BVI version', async () => {
+        let bviIsHide: string;
+
+        await driver.findElement(By.css('.bvi-open')).click();
+        await driver.wait(until.elementLocated(By.css('.bvi-body')));
+        bviIsHide = await driver.findElement(By.css('.bvi-body'))
```
Comment (VladSelikh) [inline homework_19/selenium-tests/main.spec.ts:39]: You have a repeated locator in your code: `By.css('.bvi-open')`. Please create a constant for it.
The same is for the rest of your code. Review it and create constants where it is necessary.

File: homework_19/src/config.ts [hunk 3]
```diff
@@ -0,0 +1 @@
+export const BASE_URL = 'https://tmgnews.ru/';
```
Comment (VladSelikh) [inline homework_19/src/config.ts:1]: Please rename this file into `constants.ts`.

File: homework_16/src/index.ts (hunk 1 above)
Comment (vladimirchizhenkov) [reply homework_16/src/index.ts]: fixed.

File: homework_19/src/config.ts (hunk 3 above)
Comment (vladimirchizhenkov) [reply homework_19/src/config.ts]: fixed.

File: homework_19/selenium-tests/main.spec.ts (hunk 2 above)
Comment (vladimirchizhenkov) [reply homework_19/selenium-tests/main.spec.ts]: fixed.

File: homework_17/jest-test/calculator.spec.ts [hunk 4]
```diff
@@ -95,7 +95,7 @@ describe('Jest tests for calculator:', () => {
         });
     });
 
-    describe('Jest tests for division operation', () => {
+    describe('Jest selenium-tests for division operation', () => {
```
Comment (VladSelikh) [inline homework_17/jest-test/calculator.spec.ts:98]: Why did you change that? Please return it back.

File: homework_18/api/api.spec.ts [hunk 5]
```diff
@@ -1,7 +1,7 @@
 import superagent, { Response } from 'superagent';
 import { logger } from '../src/utils';
 
-describe('API tests:', () => {
+describe('API selenium-tests:', () => {
```
Comment (VladSelikh) [inline homework_18/api/api.spec.ts:4]: Why did you change that? Please return it back.

File: homework_17/jest-test/calculator.spec.ts (hunk 4 above)
Comment (vladimirchizhenkov) [reply homework_17/jest-test/calculator.spec.ts]: fixed.

File: homework_18/api/api.spec.ts (hunk 5 above)
Comment (vladimirchizhenkov) [reply homework_18/api/api.spec.ts]: fixed.

**Review by VladSelikh — APPROVED** [review]
